### ISSUE TITLE
Netflix Error

### ISSUE THREAD

**Issue by NextedUp** [polywock/globalSpeed#51]
Suddenly stopped working on Netflix. Functions normally on all other major streaming sites.
Happy to provide any diagnostic data.

Not sure if it will help, but I noticed that "audio rending error" is reported when trying to Youtube while a Netflix tab is open + the extension is enabled. If there is no Netflix tab, it works perfect on other sites.
Maybe Netflix pushed a change on their end?

Thanks for your help and for the great extension!

**Comment by polywock** [comment]
Thanks for the positive feedback. Netflix seems to work fine for me. I'm on Chrome 87. Will be hard for me to debug without being able to reproduce it. So, I'm just going to go on a fishing expedition. 

1. What version of Global Speed?

1. What browser (and version) do use?

1. Does this issue persists on other browsers? If not...

1. Does issue persists with all other extensions disabled (except Global Speed)? If not...

1. If on Chrome/Edge, does this issue persists on a new profile? If not...

**Comment by NextedUp** [comment]
Hey there and thanks for the fast response.

 

Not sure what the root is. It is now working perfect again. Thanks again.

If you want more detail in case it becomes useful in the future somehow (but probably not):

*	What version of Global Speed? 2.7.0
*	What browser (and version) do use? Edge87/Chrome87
*	Does this issue persists on other browsers? If not... On both
*	Does issue persists with all other extensions disabled (except Global Speed)? If not... Persisted
*	If on Chrome/Edge, does this issue persists on a new profile? If not... Didn’t think to try this.

It was probably something on my end. Only had one computer here test on. But, now it is resolved (somehow), I think it was something on my end. But, I don’t know what. Tested it alone, all other extensions disable. I didn’t think to switch profiles (didn’t know that would be potentially useful – good to know for the future, thanks). Problem persisted through a full reset.

As I put in the question to github, I updated Windows to this:

Edition  Windows 10 Pro

Version 20H2

Installed on         ‎4/‎18/‎2020

OS build               19042.662

Experience         Windows Feature Experience Pack 120.2212.551.0

 

After the update reset, Netflix was working again with the extension.

 

Sorry for the false alarm but thanks for responding fast.

 

Again, great extension. I appreciate it. Very helpful listening to slow lecturers with the added utility of allowing me to procrastinate at 3x speed. I’d be happy to donate if you/your group has a site setup – this extension has definitely been worth it.
